skimage/data/__init__.py

```diff
@@ -78,7 +247,7 @@ def camera():
     camera : (512, 512) uint8 ndarray
         Camera image.
     """
-    return load("camera.png")
+    return load("data/camera.png")
```

Comment:
prefixing things with `data` allows all datasets to be relative to the root `skimage` directory. this means that we can now put any file we want in pooch.